### PR TITLE
Add v3 of EKS Infrastructure Monitoring Solution

### PR DESCRIPTION
**Pull request by Brandon-Kimberly**
*Description of changes:*
- Add alerting rules
- Add fleet monitoring dashboard
  - Minor changes to ensure the solution can be deployed to multiple clusters
- Add logs extraction
- Add logs dashboard

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: solutions/oss/eks-infra/v3.0.0/iac/cdk.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        "GRAFANA_LOGS_DASH_URL": "https://raw.githubusercontent.com/aws-observability/observability-best-practices/main/solutions/oss/eks-infra/v2.0.0/grafana-dashboards/infrastructure/logs.json",
+        "GRAFANA_APISERVER_BASIC_DASH_URL": "https://raw.githubusercontent.com/aws-observability/observability-best-practices/main/solutions/oss/eks-infra/v2.0.0/grafana-dashboards/apiserver/apiserver-basic.json",
+        "GRAFANA_APISERVER_ADVANCED_DASH_URL": "https://raw.githubusercontent.com/aws-observability/observability-best-practices/main/solutions/oss/eks-infra/v2.0.0/grafana-dashboards/apiserver/apiserver-advanced.json",
+        "GRAFANA_APISERVER_TROUBLESHOOTING_DASH_URL": "https://raw.githubusercontent.com/aws-observability/observability-best-practices/main/solutions/oss/eks-infra/v2.0.0/grafana-dashboards/apiserver/apiserver-troubleshooting.json"
+      },
+      "kustomizations": [
+        {
+          "kustomizationPath": "./solutions/oss/eks-infra/v2.0.0/infrastructure"
```
Comment (ropholuo) [inline solutions/oss/eks-infra/v3.0.0/iac/cdk.json:45]: I think it should be pointing to v3 infra?

File: solutions/oss/eks-infra/v3.0.0/iac/cdk.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        },
+        {
+          "kustomizationPath": "./solutions/oss/eks-infra/v2.0.0/apiserver"
```
Comment (ropholuo) [inline solutions/oss/eks-infra/v3.0.0/iac/cdk.json:48]: Same here?

**Review by ropholuo — CHANGES_REQUESTED** [review]
Possible typo. KustomizationPath should be pointing to v3 repo.

File: solutions/oss/eks-infra/v3.0.0/iac/cdk.json (hunk 2 above)
Comment (Brandon-Kimberly) [reply solutions/oss/eks-infra/v3.0.0/iac/cdk.json]: Fixed !

File: solutions/oss/eks-infra/v3.0.0/iac/cdk.json (hunk 1 above)
Comment (Brandon-Kimberly) [reply solutions/oss/eks-infra/v3.0.0/iac/cdk.json]: Fixed!

**Comment by Brandon-Kimberly** [comment]
This is 100% functional and ready for SME testing!

**Review by mhausenblas — APPROVED** [review]
Thanks and LGTM!

**Pull request merged**